sdk/react/controlBar.js

```diff
@@ -59,7 +59,11 @@ class ControlBar extends Component {
 
   getDurationString = () => {
     if (this.props.live) {
-      return null;
+      if (this.props.live.isLive) {
```

Comment:
Could you check this line with linter? I see that we can destruct props.

Reply:
Done